library/pk_wrap.c

```diff
@@ -1418,13 +1421,16 @@ static int rsa_alt_check_pair(const void *pub, const void *prv,
     size_t sig_len = 0;
     int ret = MBEDTLS_ERR_ERROR_CORRUPTION_DETECTED;
 
-    if (rsa_alt_get_bitlen(prv) != rsa_get_bitlen(pub)) {
+    mbedtls_rsa_context *pub_pk = mbedtls_pk_rsa(*((mbedtls_pk_context *) pub));
```

Comment:
Ah, I see now why you changed `pk_rsa()`. Though looking at `rsa_alt_get_bitlen()`, it accepts `void *`, not a `mbedtls_rsa_context *`.